src/lib/builders/slider/create.ts

```diff
@@ -123,11 +123,13 @@ export const createSlider = (props?: CreateSliderProps) => {
 
 	// Elements
 	const root = builder(name(), {
-		stores: [disabled, orientation],
-		returned: ([$disabled, $orientation]) => {
+		stores: [disabled, orientation, dir],
+		returned: ([$disabled, $orientation, $dir]) => {
 			return {
+				dir: $dir,
 				disabled: disabledAttr($disabled),
-				'aria-disabled': ariaDisabledAttr($disabled),
+				'aria-disabled': disabledAttr($disabled),
+				// TODO: Is aria-orientation missing here?
```

Comment:
It is!

Reply:
So, about that. I did some research and apparently the `aria-disabled` and `aria-orientation` attributes should be on the thumb, not the root. That's a breaking change.

Reply:
No worries, we can change these.  We can keep the data-attributes on the root as well though